imaging/ml/ml_codelab/setup.py

```diff
@@ -11,22 +11,57 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# 
+# This setup.py is used to setup dependencies and environment in Dataflow instances used for data preprocessing
```

Comment:
Hi Egor, can you add some details about why so much had to change to get it to work with Dataflow now?

Reply:
because python3 dataflow + tensorflow 1.9 incompatible, due incorrect or depricated dependency to binaries with crushing on run. we had this discussion with @codezizo  and @mfomitchev on email, if you interested in details and whole discussion we may forward it to you?

Reply:
can yo forward me that email thanks

Reply:
well, i need your email to do that:)